src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,24 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Color color;
+
+    public ColorSupplier() {
+        int index = new Random().nextInt(Color.values().length);
+        this.color = Color.values()[index];
```

Comment:
random color should be created each time when method `getRandomColor` is being called, not only once in constructor